### PR TITLE
All bitbucket updates

### PR DESCRIPTION
**Pull request by sbang002**
Updates from bitbucket

### CODE REVIEW

File: platform/android/goldengate/GoldenGateBindings/src/main/cpp/coap/jni_gg_coap_client_block.cpp [hunk 1]
```diff
@@ -182,14 +182,23 @@ static GG_Result CoapEndpoint_CancelResponseFor_Blockwise(void *_args) {
             GG_Result result = GG_CoapEndpoint_CancelBlockwiseRequest(
                     args->responseListener->endpoint,
                     args->responseListener->request_handle);
+<<<<<<< HEAD
```
Comment (ShawnW858) [inline platform/android/goldengate/GoldenGateBindings/src/main/cpp/coap/jni_gg_coap_client_block.cpp:185]: clean up the merge conflict

File: platform/android/goldengate/GoldenGateBindings/src/main/cpp/coap/jni_gg_coap_client_block.cpp [hunk 2]
```diff
@@ -182,14 +182,23 @@ static GG_Result CoapEndpoint_CancelResponseFor_Blockwise(void *_args) {
...
             if (GG_SUCCEEDED(result)) {
                 CoapEndpoint_OnResponseCompleteCleanup_Blockwise(args->responseListener);
             }
+=======
```
Comment (ShawnW858) [inline platform/android/goldengate/GoldenGateBindings/src/main/cpp/coap/jni_gg_coap_client_block.cpp:189]: clean up the merge conflict

File: platform/android/goldengate/GoldenGateBindings/src/main/cpp/coap/jni_gg_coap_client_block.cpp [hunk 3]
```diff
@@ -182,14 +182,23 @@ static GG_Result CoapEndpoint_CancelResponseFor_Blockwise(void *_args) {
...
+
+            CoapEndpoint_FreeResponseObject(args->responseListener);
+>>>>>>> bitbucket/master
             return result;
         }
         return GG_ERROR_INVALID_STATE;
     } else {
+<<<<<<< HEAD
```
Comment (ShawnW858) [inline platform/android/goldengate/GoldenGateBindings/src/main/cpp/coap/jni_gg_coap_client_block.cpp:197]: clean up the merge conflict

File: platform/android/goldengate/GoldenGateBindings/src/main/cpp/coap/jni_gg_coap_client_block.cpp [hunk 4]
```diff
@@ -604,33 +610,48 @@ Java_com_fitbit_goldengate_bindings_coap_CoapEndpoint_responseForBlockwise(
...
+    jobject thiz,
+    jlong _response_listener,
+    jboolean _canceled
+>>>>>>> bitbucket/master
 ) {
     ResponseListenerBlockwise *response_listener = (ResponseListenerBlockwise *) (intptr_t) _response_listener;
     GG_ASSERT(response_listener);
 
     CancelResponseForBlockwiseArgs args = {
+<<<<<<< HEAD
```
Comment (ShawnW858) [inline platform/android/goldengate/GoldenGateBindings/src/main/cpp/coap/jni_gg_coap_client_block.cpp:648]: clean up the merge conflict

**Comment by ShawnW858** [comment]
is https://github.com/Fitbit/golden-gate/pull/62 a duplication of this PR?

File: platform/android/goldengate/GoldenGateBindings/src/main/cpp/services/jni_gg_coap_client_service.cpp [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
-#include <logging/jni_gg_logging.h>
-#include <util/jni_gg_native_reference.h>
-
+#include "platform/android/goldengate/GoldenGateBindings/src/main/cpp/jni_gg_loop.h"
+#include "platform/android/goldengate/GoldenGateBindings/src/main/cpp/coap/jni_gg_coap_common.h"
+#include "platform/android/goldengate/GoldenGateBindings/src/main/cpp/logging/jni_gg_logging.h"
+#include "platform/android/goldengate/GoldenGateBindings/src/main/cpp/util/jni_gg_native_reference.h"
+#include "xp/common/gg_common.h"
+#include "xp/services/coap_client/gg_coap_client_service.h"
+>>>>>>> bitbucket/master
```
Comment (ShawnW858) [inline platform/android/goldengate/GoldenGateBindings/src/main/cpp/services/jni_gg_coap_client_service.cpp:10]: remove this merge conflict

**Review by ShawnW858 — APPROVED** [review]
please make sure the basic functionality of iOS and Android gg host apps still work fine.

**Comment by sbang002** [comment]
> 

Successfully built application and ran it on physical pixel 5A

**Pull request merged**